### PR TITLE
feat: make quiz call to action clickable and style

### PR DESCRIPTION
**Pull request by edpe**
https://trello.com/c/tJLAfPk2/179-make-quiz-clickable-and-style-v-ee

### CODE REVIEW

**Review by rosejbon — APPROVED** [review]
Looks great 🥇

File: src/Components/QuizEncouragement.tsx [hunk 1]
```diff
@@ -2,28 +2,40 @@ import Image from 'next/image'
...
     font-weight: bold;
     width: 90%;
+    color: ${(props) => props.theme.colours.blue};
   }
 `
 
 const ImageContainer = styled.div`
   bottom: -1rem;
-  height: 10rem;
+  height: 7rem;
```
Comment (Nikomus) [inline src/Components/QuizEncouragement.tsx:38]: Person squashed? 😢

File: src/Components/QuizEncouragement.tsx [hunk 2]
```diff
@@ -36,36 +48,39 @@ const Content = styled.div`
...
 
 export const QuizEncouragement = (): JSX.Element => {
   return (
-    <QuizCard>
-      <Content>
-        <h2>Want more personalised results?</h2>
-        <VerticalSpacing size={1} />
-        <Link href={`/quiz`} passHref>
+    <Link href={`/quiz`} passHref>
+      <QuizCard aria-label="Take our quiz">
```
Comment (Nikomus) [inline src/Components/QuizEncouragement.tsx:69]: Adding an aria-label here will hide any of the internal text (Want more personalised results?) from a screen reader, is this intentional?

**Review by Nikomus — APPROVED** [review]
Looks clickable! 🙂

File: src/Components/QuizEncouragement.tsx (hunk 2 above)
Comment (edpe) [reply src/Components/QuizEncouragement.tsx:69]: Noooooo

**Pull request merged**